graph_logic/constants.py

```diff
@@ -1,4 +1,7 @@
 from collections import defaultdict
+import json
+from os import listdir
```

Comment:
This seems unintentional

Reply:
As hints are treated as part of the inventory(?) in new logic, the easiest way to determine the maximum number of hints is to read each distro and check the one with the largest and calculate it based on that.

As far as I know, it's not possible to add EXTENDED_ITEM_NAMEs (which includes hints) to the inventory other than as constants. I suspect this can get optimized but for now I'm very keen to try it out xD

Reply:
(Github is showing me lines 2 and 3 as what you're commenting on but I'm not 100% sure that's accurate)

Reply:
I was referring to listdir